test/opt/pass_utils.h

```diff
@@ -27,8 +27,11 @@
 #ifndef LIBSPIRV_TEST_OPT_PASS_UTILS_H_
 #define LIBSPIRV_TEST_OPT_PASS_UTILS_H_
 
+#include <algorithm>
```

Comment:
Why add new header dependencies here?  Only add them if the interface described in this header needs them.